python/packages/ai/teams/ai/augmentations/monologue_augmentation.py

```diff
@@ -258,8 +258,10 @@ async def create_plan_from_response(
 
             if monologue.action.name == "SAY":
                 params = monologue.action.parameters
-                response_val = cast(str, params.get("text")) if params else ""
-                command = PredictedSayCommand(response=response_val)
+                response_val = PredictedSayCommand(
+                    response=Message(role="assistant", content=params.get("text")) if params else ""
```

Comment:
this is the JS bug I fixed this morning, you will need to add `response.message.context` to this message to ensure citations work with monologue augmentation.